.github/dependabot.yml

```diff
@@ -1,23 +1,49 @@
 version: 2
 updates:
-  # Enable version updates for library itself
+  # Enable daily version updates for prod dependencies
   - package-ecosystem: "npm"
     # Look for `package.json` and `lock` files in the `root` directory
     directory: "/"
     # Check the npm registry for updates every day (weekdays)
     schedule:
       interval: "daily"
+    labels:
+      - "dependencies"
+      - "npm"
+      - "production"
+    allow:
+      - dependency-type: production
+  # Enable weekly version updates for dev dependencies
+  - package-ecosystem: "npm"
+    # Look for `package.json` and `lock` files in the `root` directory
+    directory: "/"
+    # Check the npm registry for updates every day (weekdays)
+    schedule:
+      interval: "weekly"
+    labels:
+      - "dependencies"
+      - "npm"
+      - "development"
+    allow:
+      - dependency-type: development
+
   # Enable version updates for the website tooling
   - package-ecosystem: "pip"
     # Look for `package.json` and `lock` files in the `root` directory
     directory: "/docs"
     # Check the npm registry for updates every day (weekdays)
```

Comment:
Should we update the comment to reflect the weekly checks?

Reply:
Fixed in d15f9ee31a2cb09927a87efab780ec6b0a96e478